lib/swiftype-app-search/exceptions.rb

```diff
@@ -16,9 +16,10 @@ class Forbidden < ClientException; end
   class InvalidDocument < ClientException; end
 
   class UnexpectedHTTPException < ClientException
-    def initialize(http_response)
-      @errors = []
-      super("HTTP #{http_response.code}: #{http_response.body}")
```

Comment:
It looks like the intention here was to just call `super(http_response)`. Although I do like the way that you have fallen back to `response.message` if `response_body['errors']` is missing.